controls/srg_gpos/SRG-OS-000266-GPOS-00101.yml

```
@@ -7,4 +7,5 @@ controls:
         rules:
             - accounts_password_pam_enforce_root
             - accounts_password_pam_ocredit
+            - var_password_pam_ocredit=-1
```

```suggestion
            - var_password_pam_ocredit=1
```
you should use the variable selector, which in this case is `1`

https://github.com/ComplianceAsCode/content/blob/244458731deef03ccdb146f03cd172cd8e7f4f3b/linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/var_password_pam_ocredit.var#L17